### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'rect')

### ISSUE THREAD

**Issue by fkocovski** [clauderic/dnd-kit#465]
Since the last major update I get this error message, it appears to be generated in SortableContext.tsx:57

```ts
54 |   willRecomputeLayouts,
55 | } = useDndContext();
56 | const containerId = useUniqueId(ID_PREFIX, id);
57 | const useDragOverlay = Boolean(dragOverlay.rect !== null);
     | ^  58 | const items = useMemo(
59 |   () =>
60 |     userDefinedItems.map((item) =>
```

We did not change anything on our code end, just updated the packages. Any ideas?

**Comment by aniov** [comment]
Same problem for me too.

![image](https://user-images.githubusercontent.com/21243081/135659251-9ae82e00-7750-47e4-872d-231fadcecd1d.png)

**Comment by clauderic** [comment]
You most likely have the latest version of `@dnd-kit/sortable` but an older version of `@dnd-kit/core`, make sure you upgrade both packages to their latest versions otherwise there will be incompatible changes.

**Comment by fkocovski** [comment]
As you can see from the screenshot, I have upgraded all packages to the lastest version.

![Screenshot 2021-10-04 at 16 52 58](https://user-images.githubusercontent.com/9000611/135873886-ff83204d-ca9c-4c2d-b01d-610fdf78859d.png)

`npm outdated` doesn't show any outdated dnd-kit packages. I tried deleting the `node_modules` folder and `package_lock.json` file and running `npm install` for a fresh start but still nothing.

**Comment by aniov** [comment]
I had the same package.json config as @fkocovski when I commented on this ticket.

I solved the issue by hard coding the versions in json and reinstalling:

`"@dnd-kit/core": "4.0.0",
 "@dnd-kit/sortable": "5.0.0",
 "@dnd-kit/utilities": "3.0.0",`

In the same time using @fkocovski json in a codeSand example works.

So I don't know where the problem could be.

**Comment by fkocovski** [comment]
I also could not reproduce it in sandbox, however I managed to reproduce the problem:

We had our master branch which was still to the previous version on all packages, so we had to switch between master and the feature branch with the updated packages. When switching to the new branch from master, even if we did an `npm install` it appeared to still have leftovers. We now deployed the feature via our CI/CD pipeline which does a clean install and it appears to be working.
